src/test/fakeExtensionContext.ts

```diff
@@ -54,14 +56,32 @@ export class FakeExtensionContext implements ExtensionContext {
     public static async getNew(): Promise<FakeExtensionContext> {
         const ctx = new FakeExtensionContext()
         try {
-            ExtensionDisposableFiles.getInstance().dispose()
+            TestExtensionDisposableFiles.clearInstance()
         } catch {
-            await ExtensionDisposableFiles.initialize(ctx)
+            // Ignore.
         }
+        await TestExtensionDisposableFiles.initialize(ctx)
```

Comment:
the `catch` in the old code was supposed to be a `finally`. Typescript does not (yet?) support `try...finally` without `catch`.